kong/db/migrations/core/008_150_to_200.lua

```diff
@@ -5,6 +5,7 @@ return {
     ]],
 
     teardown = function(connector)
+      assert(connector:connect_migrations())
```

Comment:
I don't think this commit is necessary: `connect_migrations` is essentially a no-op for Postgres in these functions that get the connector, so having this here is somewhat misleading; people writing new migrations reading this code might assume it is necessary and then wonder why this is here when tracking down what the code is supposed to be achieving. If anything, it would be best if the Cassandra strategy instead received a ready-to-use connector rather than having to add a boilerplate line each time.

But generally IMO the goal of having strategies is not that their implementations should look similar, but only that their interfaces match. Anyway, this is a matter of taste, feel free to ignore if you feel strongly about it.

Reply:
>  If anything, it would be best if the Cassandra strategy instead received a ready-to-use connector rather than having to add a boilerplate line each time

I'll try to change the signature so that it is `(connector, connection|coordinator)`.

> The goal of having strategies is not that their implementations should look similar, but only that their interfaces match

Yes, this change was to make the usage of interfaces look same in both, not the implementation of migrations.

Reply:
> I'll try to change the signature so that it is (connector, connection|coordinator).

If so, let's move any API changes to another PR, so that this one can make this into a 2.1.5 patch release once we reach a conclusion!

Reply:
Ok, because all this scope creep involved here, I have rebased these changes to release/2.2.0. If we still want something to `2.1.5`, we need to backport some of the changes agreed here to be backported to another pr.